### PR TITLE
Checking package only if Glog dependency is enabled(WITH_GLOG_DEPENDE…

### PR DESCRIPTION
**Pull request by anilsripadarao**
Conditional check for the Glog package.

### CODE REVIEW

**Review by rbudai98 — APPROVED** [review]

**Comment by rbudai98** [comment]
From my side looks fine, I'd wait for @SeptimiuVana's approval as well

**Review by SeptimiuVana — APPROVED** [review]

**Pull request merged**
